### PR TITLE
Add import command

### PR DESCRIPTION
**Pull request by leecaregene**
## Add `import` command
**usage:** `import <path to file>`
**example:** `import ./data/addressbook.json)` 

This command can be used to import persons from an existing save file in `json` format.

See diff for details on files added and modified.

### **Todo:**
- [x] implement `import` command
- [x] add new test cases
- [x] update user guide

### Plans for v1.2b
Extend `import` command to support importing `CSV` files exported from Google Contacts

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ImportCommand.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ * Imports data from the JSON or CSV file located at the specified path into FinBook.
+ */
+public class ImportCommand extends Command {
+
+    public static final String COMMAND_WORD = "import";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Imports data from the JSON or CSV file located at the specified path.\n"
+            + "Parameters: path to JSON or CSV file\n"
+            + "Example: " + COMMAND_WORD + " ./somewhere/addressbook.json";
```
Comment (jovitaanderson) [inline src/main/java/seedu/address/logic/commands/ImportCommand.java:22]: I feel that the example here should be consistent with our UG. 

Possible suggestion:
`"Example: " + COMMAND_WORD  + "./data.json  for .json files in the same directory as the FinBook executable"`

File: docs/UserGuide.md [hunk 2]
```diff
@@ -175,24 +176,41 @@ If your changes to the data file makes its format invalid, FinBook will discard
...
+Format: `import PATH`
+
+* Imports data from the file at the specified `PATH`
+* `PATH` can be a relative or full path
+* `PATH` must end in `.json` or `.csv`
+
+Examples:
+
+* `import ./data.json` imports data from the file `data.json` which is located in the same directory as the FinBook executable
+* `import ../data.csv` imports data from the file `data.csv` which is located one level outside the directory of the FinBook executable
```
Comment (jovitaanderson) [inline docs/UserGuide.md:195]: I feel that the example here should be consistent with our string `MESSAGE_USAGE` in `ImportCommand.java`

**Review by jovitaanderson — COMMENTED** [review]
LGTM! :thumbsup: Just some minor inconsistency with the examples in `UserGuide.md` and `ImportCommand.java`, which may confuse the reader

**Comment by leecaregene** [comment]
Ok, I have made changes to `MESSAGE_USAGE` to make it consistent with the examples in the UG.

**Pull request merged**